Billing (#724)
neo4j/docs-aura: modules/ROOT/pages/billing.adoc

[[aura-Billing]]
= Billing
:description: Consumption reporting allows customers to monitor their billing and credit consumption.

label:Aura-DB-Professional[]
label:AuraDB-Business-Critical[]
label:AuraDB-Virtual-Dedicated-Cloud[]

Neo4j offers cloud services through prepaid consumption plans or through pay-as-you-go plans. 
Billing is based on usage.

For prepaid consumption, credits are deducted from the available balance each month. 

> For pay-as-you-go, usage is invoiced and charged on your credit card monthly in arrears.

Reviewer: ```suggestion
For pay-as-you-go, usage is invoiced and charged on your credit card monthly in arrears.

At the top-right corner of the Billing page, there's a summary of the total amount due for the current month.
This value is shown in Aura Credits (ACU), providing a real-time estimate of your projected billing.
```

> == Consumption report
> 
> The consumption report helps track usage patterns over time to make informed resource allocation decisions.
> The usage data is also exportable.
> 
> == Required roles
> 
> Available to roles `Organzation Owner`, `Organization Admin` and `Project Admin`.
> To view the billing consumption report, a payment method must be attached to the account. 

Reviewer: ```suggestion

```

> == Add new payment information
> 
> If there's no billing method attached, in a project, when *Billing* is selected, there is a call to action to *Add new payment information*.

Reviewer: ```suggestion
```
This is self explanatory.

> == Monthly summary
> 
> At the top-right corner of the Billing page, there's a summary of the total amount due for the current month. 
> This value is shown in Aura Credits (ACU), providing a real-time estimate of your projected billing.

Reviewer: ```suggestion

```
Moved to the top instead.

== Consumption report

The consumption report, accessible in the Billing section, provides real-time insights into resource usage for the current project, including both running and paused states.
It displays RAM usage in GB-hours and the equivalent cost in prepaid credits.
You can also Export a CSV file containing usage in a selected period.

> == Tabs: Instances and Sessions

Reviewer: ```suggestion
The report is available to the following roles: `Organization Owner`, `Organization Admin`, and `Project Admin`.
To view the billing consumption report, a payment method must be attached to the account.

=== Instances and Sessions
```

> There's two tabs for reviewing consumption by type:

Reviewer: ```suggestion
There are two tabs for reviewing consumption by type:
```

=== Units of measure

Usage is displayed in the specified unit of measure.
GB-hours usage is calculated by multiplying the number of hours a database is running (whether actively used or not) by the memory size in gigabytes (GB).

> == Filters

Reviewer: ```suggestion
=== Filters
```
